packages/contracts/README.md

```diff
@@ -21,7 +21,7 @@ npm install @eth-optimism/contracts
 Within your contracts:
 
 ```solidity
-import { SomeContract } from "@eth-optimism/contracts/SomeContract.sol";
+import { SomeContract } from "@eth-optimism/contracts/dist/contracts/PATH/TO/CONTRACT";
```

Comment:
Has this always been the case or has there been a regression / change in the packages exports?

Reply:
This was a change, I think.